src/app/pages/system/advanced/advanced-settings.component.ts

```diff
@@ -446,6 +450,16 @@ export class AdvancedSettingsComponent implements OnInit, AfterViewInit {
           title: helptextSystemAdvanced.fieldset_sessions,
           tableConf: this.sessionsTableConf,
         },
+        {
+          title: helptextSystemAdvanced.fieldset_token,
+          id: AdvancedCardId.Token,
+          items: [
+            {
+              label: this.translate.instant('Token Lifetime'),
+              value: this.window.localStorage.getItem('lifetime'),
```

Comment:
I think having this setting in local storage is going to be confusing for the user. We have user preferences (see `preferences.actions.ts`) that you can use here.